include/sensors/core/tasks/capacitive_sensor_callbacks.hpp

```diff
@@ -45,8 +45,10 @@ struct ReadCapacitanceCallback {
         auto capacitance = convert_capacitance(
             convert_reads(polling_results[0], polling_results[1]), 1,
             current_offset_pf);
-        auto new_offset = update_offset(capacitance, current_offset_pf);
-        set_offset(new_offset);
+        // TODO (lc 1-2-2022) we should figure out a better strategy for
```

Comment:
I still don't quite get why this capdac update strategy is wrong. Plus, won't disabling the capdac entirely break the general purpose use of the sensor?

Reply:
I think that's a debug change sneaking into the commit. It's temporarily disable for debugging the other capacitance sensor issues. This change doesn't need to be part of this PR.

Reply:
I can add it back in and then leave a comment. I think @joecrabtree was/is still worried about us setting the capdac to a spurious value (if we somehow get bad data from the IC) and so we should maybe have a better filtering strategy other than just assuming the data will be good by halving our sampling rate from the sampling rate we set on the IC.

Reply:
Yes, the implementation still has room for improvement, but we can do that along with whatever other changes fix the cap sense communication issues.

Reply:
If the sample returns to the correct value, yes. However, that could falsely trigger the threshold and cause the motor to stop.